packages/openapi-generator/src/wrapper-files/fs.spec.ts

```diff
@@ -1,5 +1,5 @@
 import * as fs from 'fs';
-import { createFile } from './create-file';
+import { createFile } from './fs';
```

Comment:
Does it make sense to add a test for `copyFile`?

Reply:
added tests